src/main/kotlin/com/ppap/torimocore/usecase/ArticleUsecase.kt

```diff
@@ -1,2 +1,26 @@
 package com.ppap.torimocore.usecase
 
+import com.ppap.torimocore.domain.Article.Article
+import com.ppap.torimocore.interfaces.database.ArticleRepository
+import org.springframework.stereotype.Service
+
+interface ArticleUsecase {
+
+    /**
+     * 投稿の一覧を参照します
+     */
+    fun showList(): List<Article>
+
+    /**
+     * 投稿の詳細を参照します
+     */
+    fun showDetail(id: Long): Article
```

Comment:
```suggestion
    fun showDetail(id: Long): Article?
```

ここは nullable な気がします。

Reply:
修正しました！